exporter/doc/view_helper.go

```diff
@@ -0,0 +1,243 @@
+package doc
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/xml"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+func escape(s string) string {
+	b := new(bytes.Buffer)
+	err := xml.EscapeText(b, []byte(s))
+	if err != nil {
+		log.Fatalln(err)
+	}
+	return b.String()
+}
+
+// TimeToFormat parses time from a seconds since Epoch value, and spits out a string in the supplied format
+func timeToFormat(t int64, f string) string {
```

Comment:
Also it looks like most if not all of these functions are just copied over from `cat1_view_helper.go`  Unless there's a really good reason that I'm missing, we should only have one place where these functions are defined.